yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -774,3 +777,133 @@ func (yb *TargetYugabyteDB) recordEntryInDB(tx pgx.Tx, batch Batch, rowsAffected
 	}
 	return nil
 }
+
+func (yb *TargetYugabyteDB) GetDebeziumValueConverterSuite(isStreamingMode bool) map[string]func(string) (string, error) {
+	converterValueSuite := make(map[string]func(string) (string, error))
+	converterValueSuite["io.debezium.time.Date"] = func(columnValue string) (string, error) {
+		epochDays, err := strconv.ParseUint(columnValue, 10, 64)
+		fmt.Printf("epochDays: %v", epochDays)
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch seconds: %v", err)
+		}
+		epochSecs := epochDays * 24 * 60 * 60
+		return time.Unix(int64(epochSecs), 0).Local().Format(time.DateOnly), nil
+	}
+	converterValueSuite["io.debezium.time.Timestamp"] = func(columnValue string) (string, error) {
+		epochMilliSecs, err := strconv.ParseInt(columnValue, 10, 64)
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch milliseconds: %v", err)
+		}
+		epochSecs := epochMilliSecs / 1000
+		return time.Unix(epochSecs, 0).Local().Format(time.DateTime), nil
+	}
+	converterValueSuite["io.debezium.time.MicroTimestamp"] = func(columnValue string) (string, error) {
+		epochMicroSecs, err := strconv.ParseInt(columnValue, 10, 64)
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch microseconds: %v", err)
+		}
+		epochSeconds := epochMicroSecs / 1000000
+		epochNanos := (epochMicroSecs % 1000000) * 1000
+		microTimeStamp, err := time.Parse(time.RFC3339Nano, time.Unix(epochSeconds, epochNanos).Local().Format(time.RFC3339Nano))
```

Comment:
I don't understand the RFC3339Nano but this is micro timestamp, so maybe it should be RFC3339Micro?

Reply:
this `RFC3339Nano` format is used to format the time with Nano seconds in timestamp, but there is not anything as such `RFC3339Micro `, so I had to use the nano one available.

Reply:
I have used a Micro secs time format in MicroTime one, but at that time when I tried that similar format here in timestamp it didn't. I will add a todo and check this later.